### DISCUSSION TITLE
SendGrid -> Paste

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2530] [Office hours] [answered]
https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049

**Comment by kelseycavitt** [comment, marked as answer]
@ceyerkes Thanks for attending OH!

[Here’s a link to the recording of our chat.](https://twilio.slack.com/archives/CBEQHHDLP/p1657818425389009)

We see a good start with your beginning phases of migrating from Sendgrid legacy to Paste. I’ve broken down our conversation into a few sections below.

**Icons**
For your needed icons, those are currently planned for our next release. We’ll update the discussion thread once they’re ready to be used.

**Table**
- Column breakdown: 
Your initial idea to move content into their own columns is something we definitely support! Having content in a single column does not allow it to be sortable. From an accessibility perspective, breaking the content into multiple columns places less cognitive load on the user. Having the columns contain less or a single piece of information is a good practice.

- Icons alone: 
We also discussed using a badge+icon for expressing the Single Send Type vs. an icon alone, this makes it more quickly obvious to the user what the type is without having to interpret on their own.

- Additional columns: 
If you wanted to be able to show more columns or even edit the columns displayed, [Fazle recently tried a show/hide columns side modal design ](https://www.figma.com/file/PJGIvUJHywtCiIGSfdnLwA/Queue-Stats?node-id=226%3A42800)that could also be an option with this design and future designs.

**Filters**
Yongjie and Kushboo h[ad started on a complex filters pattern](https://www.figma.com/file/fnTIPgdFonrT0Xxj9RzsWC/Complex-filters-pattern?node-id=1910%3A62863), but it was put on the backburner for now. They explored conditional filters and displaying several filters in a side panel (side modal). We have a side modal component in [our roadmap](https://paste.twilio.design/roadmap#2022-10-18-candidate) this year as well.

**Paste Migration**
There was actually an effort started a while ago for migrating Sendgrid to using Paste, [this doc](https://docs.google.com/document/d/1j0Bl-93j-oLt1LRR_DyOeOSSJXAGRccrBAY1bdVqvWU/edit?usp=sharing) may be helpful to review! We also had started a draft of a [Migration Strategy guidelines doc](https://docs.google.com/document/d/1-QxUaUrFTmUb7YrdOg_D5E03E7o_BGLonlU7Rxj5KwY/edit?usp=sharing) that could also be very useful to review, that shares a range of approaches to migrating to Paste.

**Continued feedback**
Please reach back out once you’re ready for more design/implementation feedback, we’re here for you!
